apps/iglidur-designer/src/app/libs/header/src/lib/footer/footer.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { FormsModule } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+
+@Component({
+  selector: 'lib-footer',
+  standalone: true,
+  imports: [
+    CommonModule,
+    MatFormFieldModule,
+    MatIconModule,
+    MatInputModule,
+    FormsModule,
+    MatButtonModule,
+  ],
+  templateUrl: './footer.component.html',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Why this?

Reply:
I wasn't able to style the Angular material elements I was using, so I found this solution that enabled me to do so.
I have not fully understood the use of the custom theming but I am working on it.

Reply:
Please this solution is not right. Encapsulation should be emulated

Reply:
Okay please.. will work on it